images/fcos-bastion-image/root/usr/lib/systemd/system-preset/98-bastion.preset

```diff
@@ -9,3 +9,4 @@ enable tang.service
 enable nfs-server.service
 enable enable-registries.service
 enable fill-network-env.service
+enable clean-registries.service
```

Comment:
This should have been 

```
enable clean-registries.timer
```

@jadhaj could you file a pr for it?

Reply:
got it. Sure